examples/basic_multicontract/contracts/.placeholder

```diff
@@ -0,0 +1 @@
+This is a placeholder file to ensure the parent directory in the git repository. Feel free to remove.
```

Comment:
Usually there is a `.gitkeep` file for this purpose, might be more familiar to a wider range of people